scripts/run-integration-test-dependencies

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
I guess you could also use this script in the gh workflow for running the integration tests. Might be nice to do that as it tests the script too..